### ISSUE TITLE
Using privitives in callback haven't invoked it.

### ISSUE THREAD

**Issue by lectricas** [square/otto#191]
So I said:
`
bus.post(228); //int`

and then tried to get it inside my Fragment and it haven't worked.

Then I changed it to String and everything became fine.

Is it ok?

**Comment by gauravsak** [comment]
@lectricas Did you get an exception in the first case? If not, what went wrong?

**Comment by lectricas** [comment]
@gauravsak  no, I haven't got any exceptions. I got just nothing. Maybe it don't know how to work with primitives

**Comment by pforhan** [comment]
This sort of thing is generally better to ask on StackOverflow.

What was your subscription method like?  I'd bet if you used an Integer as a parameter it would have worked -- I don't know if Otto is clever enough to look at unboxing values.
